core/src/main/java/nu/yona/server/analysis/entities/ActivityCommentMessage.java

```diff
@@ -76,12 +85,20 @@ public static ActivityCommentMessage createThreadHeadInstance(UUID senderUserId,
 
 	public ActivityCommentMessage getSenderCopyMessage()
 	{
-		return this.senderCopyMessage;
+		return (this.senderCopyMessageList == null || this.senderCopyMessageList.isEmpty()) ? null
```

Comment:
Suggestion: extract a util method `SingletonUtil.getValue(this.senderCopyMessageList).orElse(null)`

Reply:
This one and the next one make sense if we have this in multiple places, but currently we only have it here. If we apply it on other places too, I propose to extract it at that time, but with a different name.

Reply:
Without a util class I think it still makes sense for readability to extract a method to reflect the 'trick', if we can find a good name for it... Do you think `getSingleItem()` is a good method name to extract?